src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const bigImg = document.getElementById('largeImg');
+
+const images = document.querySelectorAll('.list-item__link');
```

Comment:
Make sure that there are elements with the class 'list-item__link' in your HTML. Otherwise, the 'images' NodeList will be empty, and the loop will not execute.